packages/router-generator/tests/generator/config-json/configured-routes-path/__root.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { Outlet, createRootRoute } from '@tanstack/react-router'
+
+export const Route = createRootRoute({
+  component: () => (
+    <React.Fragment>
+      <div>Hello "__root"!</div>
+      <Outlet />
+    </React.Fragment>
+  ),
+})
```

Comment:
Just using a very simple route configuration because not really concerned about the routes themselves.